gui.js

```diff
@@ -329,6 +329,26 @@ IDE_Morph.prototype.openIn = function (world) {
     */
     function interpretUrlAnchors() {
         var dict;
+	function doDictFlags() {
+            if (dict.editMode) {
+                myself.toggleAppMode(false);
+            } else {
+                myself.toggleAppMode(true);
+            }
+
+            if (!dict.noRun) {
+                myself.runScripts();
+            }
+
+            if (dict.hideControls) {
+                myself.controlBar.hide();
+                window.onbeforeunload = nop;
```

Comment:
this is one of the changes I've made some time ago. I don't understand why it appears in this chageset. Something is wrong about the version diffing here. I share your feeling that it should be just ```null``` or even ```delete window.onbeforeunload;```, but - strangely - when I tried those I got errors.